cmd/common/frontends/legacy/legacy.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2022-2023 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package legacy
```

Comment:
I don't think it is a proper name for it. For me, the word "legacy" sounds more like an old code that has been superseded, and we are keeping it just for backward compatibility. But I don't think it is the case. We don't even support any other frontend nowadays. Or maybe I misunderstand the word "legacy"? Maybe "terminal" or "console"?

Reply:
Yeah, agree. I had a "newer" one when developing (and renamed this one), but discarded it. I think "console" would be a much better fit.